lib/pennyworth/commands/build_base_command.rb

```diff
@@ -67,10 +67,11 @@ def base_image_create(image, tmp_dir)
     end
 
     def build_veewee(image)
-      Cheetah.run "veewee", "kvm", "build", image, "--force", "--auto"
-      Cheetah.run "veewee", "kvm", "halt", image
+      bundle_prefix = ["bundle", "exec"] if File.exists?("Gemfile")
```

Comment:
`File.exists?` is deprecated in favor of `File.exist?`.<br>Use `%w` or `%W` for array of words.